src/firefly/js/ui/SearchPanel.jsx

```diff
@@ -41,7 +42,7 @@ export function SearchPanel({style={}, initArgs={}}) {
     if (flow === 'vertical') {
         const sideBar = <SideBar {...{activeSearch, groups}}/> ;
         return (
-            <div>
+            <div style={{width: '100%', height: '100%'}}>
```

Comment:
Same concern as Menu.css.  These are generic code.  Are you sure it is needed by everyone.

Reply:
Hey @loitly, so you're right, these are not needed by the other apps. But mainly for router apps (the idea being they end up using the entire screen's width and height for the MOC background). I found that adding these doesn't affect other apps, but nonetheless, it still applies these styles to other apps, which is why I initially had conditional styles for router based apps. If I don't add the style here and in menu.css, then this is what euclid, for example, looks like (similar to Wise/SHA, etc.): 

<img width="1425" alt="euclid_ss3" src="https://github.com/Caltech-IPAC/firefly/assets/110492183/e83ab242-1e09-4700-8556-84f798406625">

If we don't want to add style in this way or conditionally if it's router based app, do you have a suggestion on how I could approach this?

Reply:
Actually @loitly, I was able to get rid of this change from menu.css at least! 

(based on your suggestion in irsa-ife PR) By adding the following to euclid.css instead: 

```
.SearchPanel {
    height: 100%;
    width: 100%;
}
```

But I think I still need to add `style={{width: '100%', height: '100%'}}` to the div in SearchPanel.jsx, becuase I tinkered with it a lot, and without adding it there, we end up with something like this: 

<img width="1431" alt="euclid_ss2" src="https://github.com/Caltech-IPAC/firefly/assets/110492183/e1593d6d-4e03-4502-8e80-1307632444fe">

Reply:
I see what the problem is.  I would add `className='SearchPanel-Box` to line: 36, 46, and 61.  This allow other to override as needed.  Putting fixed style without the ability to override may present an issue later.

Reply:
@loitly just to confirm, you mean I should add this className to the existing div elements right? (which would be lines 35, 45 and 60?) so it would look like, for example, for lines 45-47: 

```
<div className='SearchPanel__box'>
                {title && <h2 style={{textAlign: 'center'}}>{title}</h2>}
                <div className='SearchPanel' style={style}>
```

Reply:
No.  Here's an example...
```
            <div className='SearchPanel-Box'>
                {title && <h2 style={{textAlign: 'center'}}>{title}</h2>}
                <div className='SearchPanel' style={isUsingRouter ? routerStyle : style}>
                    {sideBar}
                    {searchItem &&
                    <div className='SearchPanel__form'>
                        <SearchForm searchItem={searchItem}  initArgs={initArgs}/>
                    </div>
                    }
            </div>

```

Reply:
wait, I'm confused, isn't that the same? (the div that `className='SearchPanel-Box'` is being added to?)

Reply:
@loitly can you see the latest commit here? I removed the unused css and added style for `SearchPanel-Box` (that's in the irsa-ife PR) and over here I added the` SearchPanel-Box` className in 3 places. Does that look right?